File: app/src/main/java/org/apache/fineract/data/models/product/BalanceRange.java
package org.apache.fineract.data.models.product;

/**
 * @author Rajan Maurya
 *         On 20/07/17.
 */

public class BalanceRange {

    private Double minimum;
    private Double maximum;

    public Double getMinimum() {
        return minimum;
    }

    public void setMinimum(Double minimum) {
        this.minimum = minimum;
    }

    public Double getMaximum() {
        return maximum;
    }

    public void setMaximum(Double maximum) {
        this.maximum = maximum;
    }
}

File: app/src/main/java/org/apache/fineract/data/models/product/BalanceRange.kt
package org.apache.fineract.data.models.product

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

/**
 * @author Rajan Maurya
 * On 20/07/17.
 */

@Parcelize
data class BalanceRange(
        @SerializedName("minimum") var minimum: Double? = null,
        @SerializedName("maximum") var maximum: Double? = null
) : Parcelable
